docs/langref/relay/expressions.rst

```diff
@@ -0,0 +1,162 @@
+===========
+Expressions
+===========
+
+The Relay IR is a pure, expression oriented language, with a dataflow fragment
```

Comment:
"Expression-oriented" should be hyphenated (compound adjective)

Additionally, I think in the next line it should say "Although Relay's representation is an abstract syntax tree" to clarify what exactly we mean by tree (i.e., not a tree in the sense of an acyclic graph)